FAQ.md

```diff
@@ -44,11 +44,15 @@ TiDB is self-healing. All of the three components, TiDB, TiKV and PD, can tolera
 
 #### How is TiDB strongly consistent?
 
-TiDB uses the [Raft consensus algorithm](https://raft.github.io/) to ensure consistency among multiple replicas. At the bottom layer, TiDB uses a model of replication log + State Machine to replicate data. For the write requests, the data is written to a Leader and the Leader then replicates the command to its Followers in the form of log. When the majority of nodes in the cluster receive this log, this log is committed and can be applied into the State Machine. TiDB has the latest data even if a minority of the replicas are lost.
+TiDB implements the Repeatable Read isolation in the Snapshot Isolation level. It does so by using the [Raft consensus algorithm](https://raft.github.io/) to ensure consistency among multiple replicas.
```

Comment:
Perhaps something like:
```suggestion
TiDB implements Snapshot Isolation consistency, which it advertises as `REPEATABLE-READ` for compatibility with MySQL. Data is redundantly copied between TiKV nodes using the [Raft consensus algorithm](https://raft.github.io/) to ensure recoverability should a node failure occur.
```
The point I am making here is that raft doesn't provide consistency unless there is a failure event.  It is the transactional protocol on top of Raft that provides it.  Raft provides recoverability.